README.md

```diff
@@ -1,8 +1,67 @@
+# JSON Compilation Database Format Specification
+
+This document describes a format for specifying how to replay single link commands independently of the build system.
+
+## Background
+
+Like compilation commands, link commands are needed for code analysis, for example, look into simple C/C++ code:  
```

Comment:
Tell that there are compilation commands and that they are needed for code analysis, and provide a link to clang documentation. Then say that for advanced code analysis you also need link commands database. And then provide an example